janken.py

```diff
@@ -1 +1,19 @@
 
+you_count = 0
+
+for a in range(3):
+    n = 0
+    you = 0
+    enemy = r.randrange(1,4)
+    while n<1:
+        you = int(input('-----------\nグー : 1\nチョキ : 2\n パー : 3\nあなたの手 >>'))
+        if you > 3 :
+            print('半角数字1~3')
+        elif you == enemy:
+            print('あいこ\n\n-----------')
+            enemy = r.randrange(1,4)
+        else:
+            n = 1
+    jugdement(you,enemy)
```

Comment:
```suggestion
    jugdement(you,enemy)
```
ここの文は無くなっちゃっても大丈夫だと思います！